bigbone/build.gradle

```diff
@@ -2,13 +2,48 @@ plugins {
     id 'social.bigbone.bigbone-library'
 }
 
+sourceSets {
+    integrationTest {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
+    }
+}
+
+configurations {
+    integrationTestImplementation.extendsFrom implementation
+    integrationTestRuntimeOnly.extendsFrom runtimeOnly
+}
+
 dependencies {
     api libs.okhttp
     api libs.gson
     implementation libs.kotlin.jdk8
 
-    testImplementation libs.junit.jupiter
+    testImplementation libs.junit.jupiter.api
     testImplementation libs.kluent
     testImplementation libs.mockk
     testImplementation libs.mockk.dsl
+    integrationTestImplementation libs.junit.jupiter.api
+    integrationTestImplementation libs.junit.jupiter.engine
+    integrationTestImplementation libs.junit.platform.suite.engine
+    integrationTestImplementation libs.testcontainers
```

Comment:
testcontainers dependencies are not needed atm, as we just rely on pure docker compose for the moment.... Will clean this up.

Reply:
Fixed.